Sources/ApolloCodegenLib/Templates/InputObjectTemplate.swift

```diff
@@ -12,20 +12,21 @@ struct InputObjectTemplate: TemplateRenderer {
 
   var template: TemplateString {
     let (validFields, deprecatedFields) = filterFields(graphqlInputObject.fields)
+    let memberAccessControl = accessControlModifier(target: target, definition: .member)
```

Comment:
Reduce into a function on `TemplateRenderer` to only accept scope and take in target from each template.